crates/sdk/Cargo.toml

```diff
@@ -97,6 +97,7 @@ borsh-ext.workspace = true
 circular-queue.workspace = true
 data-encoding.workspace = true
 derivation-path.workspace = true
+duration-str = "0.10.0"
```

Comment:
let's declare this version in the workspace `Cargo.toml` - I think only exception to this should be the `crates/apps` for binaries specific deps

Reply:
there are couple more in here too

Reply:
fixed in a8c3d66b9b557ed4e73156615632905a247a3c04